components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m

```diff
@@ -28,13 +28,9 @@ file conf/11-x11-synaptics.fdi path=etc/hal/fdi/policy/10osvendor/11-x11-synapti
 
 file path=usr/include/xorg/synaptics-properties.h
 file path=usr/lib/$(MACH64)/pkgconfig/xorg-synaptics.pc
-file path=usr/lib/pkgconfig/xorg-synaptics.pc
 file path=usr/lib/xorg/$(MACH64)/synclient mode=0555
 file path=usr/lib/xorg/$(MACH64)/syndaemon mode=0555
 file path=usr/lib/xorg/modules/input/$(MACH64)/synaptics_drv.so
-file path=usr/lib/xorg/modules/input/synaptics_drv.so
```

Comment:
on Linux synclient/syndaemon live in usr/bin, should we move at least synclient there?

Reply:
Sure I can do it.

Reply:
Done